model/database/registry.yaml

```diff
@@ -543,7 +543,12 @@ groups:
         stability: experimental
         brief: Account or request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
         examples: ["Eventual", "ConsistentPrefix", "BoundedStaleness", "Strong", "Session"]
-
+      - id: db.cosmosdb.regions_contacted
+        type: string[]
+        stability: experimental
+        brief: Comma separated list of regions contacted by the Azure Cosmos DB operation.
```

Comment:
does order matter?

does case-sensitivity matter?

Reply:
order doesn't matter and case-sensitivity, ideally should not matter as I would never see _West US 2_ and _WEST US 2_ together. and both would be representing same region.

Reply:
does it make sense to normalize these in any way, so that e.g. different orders don't create different time series (different attribute sets)?

Reply:
also can we normalize the values in the list and list all of the supported values and/or how to normalize it?

The goal is to have cross-language consistency and alignment with other areas (e.g. REST API values, some existing Azure location list, etc).

Reply:
@sourabh1007 is the list ordered in the same sequence that the regions were actually contacted by the SDK? If that's the case I think we should leave order as is, if the order can be "random" then we should alphabetize or some other standarization

Reply:
it is random order list. we can sort it.

Reply:
Let's document the format of the region. As we discussed in the https://github.com/open-telemetry/semantic-conventions/pull/1543#discussion_r1831606621 is should be the `nae` or `displayName` from https://learn.microsoft.com/en-us/rest/api/subscription/subscriptions/list-locations?view=rest-subscription-2021-10-01&tabs=HTTP#location.

BTW I don't understand why `displayName` is preferred - it has spaces and case-sensitive things are always problematic in user queries. I'd use `name`